lib/BitcastUtils.cpp

```diff
@@ -818,8 +818,10 @@ bool RemoveCstExprFromFunction(Function *F) {
 
   auto CheckInstruction = [&WorkList](Instruction *I) {
     for (unsigned OperandId = 0; OperandId < I->getNumOperands(); OperandId++) {
-      if (isa<ConstantExpr>(I->getOperand(OperandId))) {
-        WorkList.push_back(std::make_pair(I, OperandId));
+      if (dyn_cast<ConstantExpr>(I->getOperand(OperandId)) ||
+          dyn_cast<ConstantVector>(I->getOperand(OperandId)) || 
+          dyn_cast<ConstantArray>(I->getOperand(OperandId))) {
```

Comment:
This adds constant data sequential handling. Is constant aggregate handling also needed?

Reply:
I guess, it won't be part of this PR. It was not part of the issues in #1223.